dolphinscheduler-ui/src/locales/modules/en_US.ts

```diff
@@ -997,6 +997,14 @@ const project = {
     mlflowProjectRepository_tips: 'github respository or path on worker',
     mlflowProjectVersion: 'Project Version',
     mlflowProjectVersion_tips: 'git version',
+    openmldb_zk_address: 'zookeeper address',
```

Comment:
we should modify system screenshots at the same time when we change the keyword from `zk` to `zookeeper`